shoop/admin/modules/products/views/forms.py

```diff
@@ -186,7 +186,15 @@ def __init__(self, **kwargs):
             self.fields["shops"].initial = [default_shop]
 
         self.file_url = self.instance.url
-        self.thumbnail = None
+
+        if self.instance and self.instance.file:
+            try:
+                thumbnail = self.instance.get_thumbnail()
+                self.thumbnail = thumbnail.url
+            except Exception:
+                self.thumbnail = None  # no thumbnail available for whatever reason
+        else:
+            self.thumbnail = None
```

Comment:
1. Originally this was my intention, somehow @akx turned my head to make this kind of solution. It's probably not what @akx was after, but he said `ProductMedia.get_thumbnail` should raise when errors occur, now it does. 
2. Agree catch is too broad, and I need to find out what are the exact Exceptions. It has been catched this way in prior implementations inside Shoop.
3. `file_url` is the url to original file, thumbnail url is for the thumbnail, if that was something you were after here, not sure I completely understand :)